src/frontends/jax/src/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if((CMAKE_COMPILER_IS_GNUCXX OR OV_COMPILER_IS_CLANG) AND CMAKE_CXX_STANDARD GREATER_EQUAL 20)
+    set(CMAKE_CXX_FLAGS "-Wno-error=deprecated ${CMAKE_CXX_FLAGS}")
+endif()
+
+ov_add_frontend(NAME jax
+                LINKABLE_FRONTEND
+                SHUTDOWN_PROTOBUF
```

Comment:
as I see, JAX frontend does not use protobuf, hence `SHUTDOWN_PROTOBUF` is not required.